test/modules/obsagent-client/docker/linux/arm64v8/base/Dockerfile

```diff
@@ -0,0 +1,8 @@
+﻿FROM arm64v8/ubuntu:18.04
+
+RUN apt-get update && apt-get install -y protoc
+
+# Add an unprivileged user account for running the module
+ARG MODULEUSER_ID=1000
+RUN useradd -u ${MODULEUSER_ID} -ms /bin/bash moduleuser
+USER moduleuser
```

Comment:
We don't need this base image. It is simpler for base image management if we share the same base image as the `generic-mqtt-tester`.

So we should delete here and in the path `.../linux/arm32v7/base/Dockerfile`